python/metatensor-core/metatensor/labels.py

```diff
@@ -956,6 +956,34 @@ def intersection_and_mapping(
 
         return Labels._from_mts_labels_t(output), first_mapping, second_mapping
 
+    def select(self, selection: "Labels") -> np.ndarray:
+        """
+        Select entries in these :py:class:`Labels` that match the ``selection``.
+
+        The selection's names must be a subset of the names of these labels.
+
+        All entries in these :py:class:`Labels` that match one of the entry in the
+        ``selection`` for all the selection's dimension will be picked. Any entry in the
+        ``selection`` but not in these :py:class:`Labels` will be ignored.
+
+        :param selection: description of the entries to select
+        :return: 1-dimensional ndarray containing the integer indices of selected
+            entries
```

Comment:
Maybe add an example as for the other methods.

Reply:
Done!